scrabble-site.rb

```diff
@@ -0,0 +1,40 @@
+require "sinatra"
+require "./lib/scrabble"
+
+class ScrabbleSite < Sinatra::Base
+
+  get "/" do
+    erb :index
+  end
+
+  get "/score" do
+    erb :score
+  end
+
+  post "/score" do
+    @word = params[:word]
+    @score = ScrabbleMod::Scrabble.score(@word)
+    erb :score
+  end
+
+  get "/multiscore" do
+    @words = []
+    @words_hash = {}
+    erb :multiscore
+  end
+
+  post "/multiscore" do
+    big_string = params[:words]
+    @words = big_string.split(",")
+    @words_hash = {}
+    @words.each do |word|
+      @words_hash[word] = ScrabbleMod::Scrabble.score(word)
```

Comment:
Good use of a hash to hold both the word data and the score associated with that word.